.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+#### What Does This PR Do?
+
+#### Description Of Task To Be Completed
+
+#### Any Background Context You Want To Provide?
+N/A
+
+#### How can this be manually tested?
+
+#### Screenshot(s)
+N/A
+
+#### What are the relevant github issues?
```

Comment:
Not really. This is just an oversight for when bugs and features are requested.
Currently we have no ID on `trello`, so in the event where issues are being reported here we can attach that. 

But for now, it's not going to be used for a while.